static/js/util/currency.js

```diff
@@ -37,7 +37,7 @@ const codesToOptions = R.compose(
 
 export const currencyOptions = codesToOptions(cc.codes());
 
-const codeToCountryName = code => iso3166.country(code).name || '';
+export const codeToCountryName = code => iso3166.country(code).name || '';
```

Comment:
this function needs to have the type of it's argument annotated, now that it's exported. if you're not familiar with Flow, it will end up looking like this:

```js
export const codeToCountryName = (code: string) => iso3166.country(code).name || '';
```